packages/block-editor/src/components/global-styles/filters-panel.js

```diff
@@ -189,15 +189,16 @@ export default function FiltersPanel( {
 
 							return (
 								<ItemGroup isBordered isSeparated>
-									<Button
+									<Item
+										as="button"
 										__next40pxDefaultSize
```

Comment:
Does it even need this 40px size handling anyway? I'm seeing that the button is already pretty much the correct size (40px minus border widths = 38px) due to [these styles](https://github.com/WordPress/gutenberg/blob/1418350eb5a1f15e109fc96af385bdd029fc7304/packages/block-editor/src/components/global-styles/style.scss#L30). I might be missing something.

```suggestion
```

Reply:
Yes I noticed it's 38px + the borders. It's incorrect anyways.
I have no strong opinions, I can remove the `__next40pxDefaultSize` tempative fix, as I mentioned it was for demoing a potential way to fix it. I'd like to prevent designers complain about an incorrect height :)